### ISSUE TITLE
AssertionError in self.stats['set_deduplicated'] in python3

### ISSUE THREAD

**Issue by leggewie** [kdeldycke/mail-deduplicate#89]
I use version 3.0.0, installed via pip3 on Ubuntu Bionic.  That means, my standard python interpreter is python 2.7.17.  That leads to the following AssertionError.

```
mdedup deduplicate ~/Maildir/.DAMAGED20200924.Usenet-Speicher/ ~/Maildir/.Usenet-Speicher/
[...]
Check that mail differences are within the limits.
╒════════════╤══════════╕
│ Mails      │   Metric │
╞════════════╪══════════╡
│ Found      │       70 │
├────────────┼──────────┤
│ Skipped    │        0 │
├────────────┼──────────┤
│ Rejected   │        0 │
├────────────┼──────────┤
│ Kept       │       70 │
├────────────┼──────────┤
│ Unique     │        0 │
├────────────┼──────────┤
│ Duplicates │       70 │
├────────────┼──────────┤
│ Deleted    │        0 │
╘════════════╧══════════╛
╒══════════════════════════════════════╤══════════╕
│ Duplicate sets                       │   Metric │
╞══════════════════════════════════════╪══════════╡
│ Total                                │       35 │
├──────────────────────────────────────┼──────────┤
│ Ignored                              │        0 │
├──────────────────────────────────────┼──────────┤
│ Skipped                              │        0 │
├──────────────────────────────────────┼──────────┤
│ Rejected (bad encoding)              │        0 │
├──────────────────────────────────────┼──────────┤
│ Rejected (too dissimilar in size)    │        0 │
├──────────────────────────────────────┼──────────┤
│ Rejected (too dissimilar in content) │        0 │
├──────────────────────────────────────┼──────────┤
│ Deduplicated                         │        0 │
╘══════════════════════════════════════╧══════════╛
Traceback (most recent call last):
  File "/home/leggewie/.local/bin/mdedup", line 11, in <module>
    sys.exit(cli())
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/leggewie/.local/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/leggewie/.local/lib/python3.6/site-packages/mail_deduplicate/cli.py", line 219, in deduplicate
    dedup.report()
  File "/home/leggewie/.local/lib/python3.6/site-packages/mail_deduplicate/deduplicate.py", line 648, in report
    self.stats['set_deduplicated'])
AssertionError
```

**Comment by kdeldycke** [comment]
This has been fixed in https://github.com/kdeldycke/mail-deduplicate/commit/53495a33cd8d11ba97a29fb890e2b527104ce1bc . It will be released with v4.0.0.

**Comment by leggewie** [comment]
Awesome, thanks!
